ansible/group_vars/all

```diff
@@ -464,7 +466,7 @@ zeroDowntimeDeployment:
   enabled: "{{ zerodowntime_deployment_switch | default(false) }}"
 
 etcd:
-  version: "{{ etcd_version | default('v3.4.33') }}"
+  version: "{{ etcd_version | default('3.5') }}"
```

Comment:
etcd supports multiarch since 3.5 version.
https://hub.docker.com/r/bitnami/etcd/

<img width="695" alt="image" src="https://github.com/user-attachments/assets/edd6cced-e73c-405e-8799-cf757c4748e6">

Reply:
I know this is stale now, but does etcd still require one to enable experimental features to run on arm?

Reply:
Ignore that, I found the answer. Looks like arm64 has first-class support now as of 3.5. Previously it required one to use the ETCD_UNSUPPORTED_ARCH environment variable.